Changes/Additions to Developers Section (#121)
archway-network/archway-docs: docs/create/guides/nft-project/dapp.mdx

title: Building the NFT dapp
---

> If this is your first time building an Archway dapp frontend, head over to the [Creating Your First dapp](../my-first-dapp/dapp.mdx) guide to learn how to setup your basic project scaffolding. Once you're serving a web page that connects to [Keplr](https://wallet.keplr.app/), you'll be ready to follow the next steps.

Reviewer: ```suggestion
If it's your first time building an Archway dapp frontend, head over to the [Creating Your First dapp](../my-first-dapp/dapp.mdx) guide to learn how to setup your basic project scaffolding. Once you're serving a web page that connects to [Keplr](https://wallet.keplr.app/), you'll be ready to follow the next steps.
```

    tokenId = '2', 
    imageUrl = "ipfs://QmY8Ei2NQkMg8rCFiUkcXH2NPWPNKMoWLBYDwBeMNsP5sr", 
    name = "Archway NFT #2", 

>     description = "Building With NFTs (and dapps)", 

Reviewer: ```suggestion
    description = "Building With NFTs (and Dapps)", 
```
